README.md

```diff
@@ -150,6 +150,13 @@ Like above, but no docker.
 
 Like above, but s4cmd instead of s3cmd
 
+## kiwicom/sentry
+
+- Base image: [`sentry:9.0.0`](https://hub.docker.com/_/sentry/)
```

Comment:
We'll forget to update this for every patch version, maybe just mention `sentry` alone?

Reply:
🆗